### DISCUSSION TITLE
How to log a text file via wandb in configuration?

### DISCUSSION THREAD

**Discussion by elichen3051** [axolotl-ai-cloud/axolotl#2222] [Q&A]
Dear Partners,

I'm a algorithm developer and often modify the source code of huggingface transformers.  Once I conduct the experiment, I want to record the source code, modules in transformers. Does axolotl support this feature?

Thanks,
Eli Chen

**Comment by NanoCode012** [comment]
Hey, unfortunately, we don't have that feature. I believe if you turn on Code Saving in wandb, it would record changes in the current git repo. However, I don't think it's able to record edited external packages like transformers when running axolotl.
